lua/catppuccin/groups/integrations/diffview.lua

```diff
@@ -0,0 +1,51 @@
+local M = {}
+
+function M.get()
+	return {
+		DiffviewDim1 = { link = "Comment" },
+		DiffviewPrimary = { fg = C.blue },
+		DiffviewSecondary = { fg = C.green },
+		DiffviewNormal = { fg = C.text, bg = O.transparent_background and C.none or C.mantle },
+		DiffviewNonText = { link = "NonText" },
+		DiffviewCursorLine = { link = "CursorLine" },
+		DiffviewWinSeparator = {
+			fg = O.transparent_background and C.surface1 or C.base,
+			bg = O.transparent_background and C.none or C.base,
+		},
+		DiffviewSignColumn = { link = "Normal" },
+		DiffviewStatusLine = { link = "StatusLine" },
+		DiffviewStatusLineNC = { link = "StatusLineNC" },
+		DiffviewEndOfBuffer = { link = "EndOfBuffer" },
```

Comment:
As they are linked in diffview by default, maybe these code can be removed? There're also code like this below and above.

Reply:
Thank you for your comment. I have removed the code that you mentioned above ❤️